src/main/drivers/pwm_output.c

```diff
@@ -94,9 +95,9 @@ void pwmOutConfig(timerChannel_t *channel, const timerHardware_t *timerHardware,
 
 #if defined(USE_HAL_DRIVER)
     if (timerHardware->output & TIMER_OUTPUT_N_CHANNEL)
-        HAL_TIMEx_PWMN_Start(Handle, timerHardware->channel);
+         HAL_TIMEx_PWMN_Start(Handle, timerHardware->channel);
```

Comment:
Please don't make unrelated changes.
It makes merging difficult later with other code changes.

Reply:
Fixed.